### PR TITLE
quick start: with GitHub Codespaces

### PR DESCRIPTION
**Pull request by amyXia1994**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [ ] N
- [X] Y 

docs related to #175 

#### 2. What is the scope of this PR (e.g. component or file name):
quick start guide CN version and gifs

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

add guide docs about kusion quick start based on GitHub Codespaces & minikube.

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Other

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [ ] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [ ] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [ ] Other

#### 6. Release note

Please refer to [Release Notes Language Style Guide](https://kusionstack.io/docs/governance/release-policy/) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Comment by howieyuen** [comment]
Can you make these PNG and GIF again? Not only the big size, but also I see your NetEase page 🤣

**Comment by amyXia1994** [comment]
> Can you make these PNG and GIF again? Not only the big size, but also I see your NetEase page 🤣

got it. I've noticed that, too and I'll update the gifs later.

**Review by SparkYuan — APPROVED** [review]
lgtm

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/user_docs/getting-started/install/cloudide.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+![minikube start](/img/docs/user_docs/getting-started/install/codespaces/minikube-start.gif)
+
+此外，我们创建的工作空间会默认展示 `appops/guestbook-frontend/base/base.k` 和 `appops/guestbook-frontend/dev/main.k` 两个文件，它们描述了 `guestbook-frontend` 应用的部署配置。可以看到应用的 `dev/main.k` 文件中声明了研发环境对应的镜像版本为 `gcr.io/google-samples/gb-frontend:v5`，而 `base/base.k` 中则描述了该应用在各环境下的通用配置——例如名为 `frontend`、通过 80 端口提供服务的服务资源。
+
+![view code](/img/docs/user_docs/getting-started/install/codespaces/view-code.gif)
+
+### Step2：使用 Kusion 发布应用
+
+minikube 启动完成后，我们即可使用 kusion 在 minikube 中拉起研发版本的 `guestbook-frontend` 应用，预览待发布应用资源与集群资源的 live-diff，确认无误后继续完成发布：
+- 在 dev 目录下的配置文件 `main.k` 中，右键点击 `Kusion: Preview Live Diff and Apply`
```
Comment (He1pa) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/user_docs/getting-started/install/cloudide.md:30]: 只有 base/prod/pre/test， 没有 dev 目录

**Review by howieyuen — DISMISSED** [review]

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/user_docs/getting-started/install/cloudide.md (hunk 1 above)
Comment (amyXia1994) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/user_docs/getting-started/install/cloudide.md:30]: fixed in: https://github.com/KusionStack/konfig/pull/79, ptal

**Review by He1pa — DISMISSED** [review]
LGTM

**Pull request merged**